src/components/structureDefSettings/complexTypesEditor/RenderComplexType.tsx

```diff
@@ -170,10 +210,13 @@ const RenderComplexType: React.FC<DetailProps> = ({
             const mapValues: {
               value: string | undefined;
               label: string | undefined;
-            }[] = [
-              { value: "--select a value--", label: "--select a value--" }
-            ];
-            item.valueSet?.forEach((values) =>
+            }[] = [];
+            if (item.min === 0)
+              mapValues.push({
+                value: "--select_a_value--",
```

Comment:
I suggest to let an empty string "" there and change l100 in StructureDefSettings